README.md

```diff
@@ -0,0 +1,73 @@
+# Release Notes Action
+
+Generate release-notes for a given version from CHANGELOG file.
+
+## Usage
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+
+  - id: release-notes
+    uses: freckle/release-notes-action@v1
+
+  - uses: actions/create-release@v1
+    id: create-release
+    with:
+      body_path: ${{ steps.release-notes.outputs.path }}
+      # ...
+```
+
+## Inputs
+
+- **file**: Path to the CHANGELOG
+
+  Default is `./CHANGELOG.md`.
+
+- **version**: Version to locate
+
+  Default is empty which means most first (assumed to be latest).
+
+- **header-regex**: Regular expression used to match headers.
```

Comment:
```suggestion
- **header-regex**: Regular expression used to match section headers
```

Should this be `version-header-regex`?

Reply:
Yea, I think `version-header-regex` is better.